src/api/spec/cassettes/Comments_with_diff/reply_comment/when_under_the_beta_program/displays_the_changes_after_adding_a_new_comment.yml

```diff
@@ -992,27 +1026,27 @@ http_interactions:
       Connection:
       - close
       Content-Length:
-      - '404'
+      - '406'
     body:
       encoding: UTF-8
       string: |
-        <directory name="package_a" rev="6" vrev="6" srcmd5="a09afa482a997030612d990e3b7733aa">
-          <entry name="README.txt" md5="2e6518277c9d443c0ae45a18ec0c0f1d" size="62" mtime="1729497327"/>
-          <entry name="package_a.changes" md5="3bf7f645ebf1daed37ac2524846a9e7c" size="14" mtime="1729496885"/>
-          <entry name="package_a.spec" md5="674078c9d96283bf095d60ffe47009a0" size="470" mtime="1729496885"/>
+        <directory name="package_a" rev="21" vrev="21" srcmd5="fafdb3cebde7662ca252456e669f6005">
```

Comment:
Sounds like you did not drop your backend before refreshing the cassette...

Reply:
Dropping the backend before refreshing the cassette? I don't understand...